pegasus/migrations/109_add_hoc_learn_activity_table.rb

```diff
@@ -0,0 +1,11 @@
+Sequel.migration do
+  change do
+    create_table(:hoc_learn_activity, charset: 'utf8') do
+      primary_key :id
+      String :session, index: true, size: 50
+      String :referer, size: 50
+      String :tutorial, index: true, null: false, size: 50
+      DateTime :created_at, index: true
```

Comment:
oooh, good point. I can't think of a reason that we'd want an updated_at, but I like the idea of adding it now just for future-proofing.